dumps/obj_fetch_non-key.c

```diff
@@ -0,0 +1,218 @@
+                                                        id = &cache.ac_entries[i];
+                                                        status = ilog_status_get(lctx, id, intent, retry);
+                                                                vos_ilog_status_get /* cbs->dc_log_status_cb */
+                                                                        rc = vos_dtx_check_availability(coh, tx_id, epoch, intent, DTX_RT_ILOG, retry);
+                                                                                dth = vos_dth_get(cont->vc_pool->vp_sysdb);
+                                                                                if (dtx_is_committed(entry, cont, epoch))
+                                                                        /* rc == ALB_AVAILABLE_CLEAN */
+                                                        /* status == ILOG_COMMITTED */
+                                                        entries->ie_info[entries->ie_num_entries].ii_removed = 0;
+                                                        entries->ie_info[entries->ie_num_entries++].ii_status = status;
+                                                /* entries->ie_num_entries == 1 */
+                                                priv->ip_rc = rc; /* 0 */
+#endif
+                                                vos_parse_ilog(info, epr, bound, &punch); TG
+                                                        ilog_foreach_entry_reverse(&info->ii_entries, &entry) {
+                                                                /* entry.ie_status != ILOG_REMOVED */
+                                                                /* !vos_ilog_punched(&entry, punch)) */
+                                                                entry_epc = entry.ie_id.id_epoch; /* 5 */
+                                                                /* if (entry_epc > epr->epr_hi) { *//* {epr_lo = 0, epr_hi = 6} */ ! 5 > 6
+                                                                /* if (entry.ie_status == -DER_INPROGRESS) FALSE */
+                                                                /* !vos_ilog_punch_covered(&entry, &info->ii_prior_any_punch) */
+                                                                /* if (entry.ie_status == ILOG_UNCOMMITTED) == FALSE /* ie_status == -138768134 */ */
+                                                                if (entry.ie_id.id_epoch > info->ii_uncommitted) /* 9975412875014583296 > 0 */
+                                                                        info->ii_uncommitted = 0;
+                                                                /* entry.ie_status == ILOG_COMMITTED */
+                                                                /* if (ilog_has_punch(&entry)) */
+                                                                info->ii_create = entry.ie_id.id_epoch; /* 9975412875014583296 */
```

Comment:
I understand this value is just an example. You do not observe it consistently between different runs, do you?